zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2717,6 +2766,18 @@ components:
           description: All the variables visible in the root scope.
           type: object
 
+    DeleteResourceRequest:
+      type: object
+      nullable: true
+      properties:
+        operationReference:
+          description: |
+            A reference key chosen by the user and will be part of all records resulting from this operation.
```

Comment:
```suggestion
            A reference key chosen by the user that will be part of all records resulting from this operation.
```

Reply:
@nicpuppa, this is still `and` instead of `that` in the file. Have you accidentally resolved this already or am I overlooking something? 😄